uzdevums_6.py

```diff
@@ -0,0 +1,11 @@
+print("Ievadiet skaitli ar kuru velaties parbaudit vai sakitlis dalas ar 5 vai 7 :")
+n=input()
```

Comment:
Input komandai var padot lietotājam izvadāmu tekstu. Līdz ar to šo var izdarīt vienā rindā:
```suggestion
n = input("Ievadiet skaitli ar kuru velaties parbaudit vai sakitlis dalas ar 5 vai 7 :")
```